app/models/navigation_link.rb

```diff
@@ -0,0 +1,8 @@
+class NavigationLink < ApplicationRecord
+  URI_REGEXP = URI::DEFAULT_PARSER.make_regexp(%w[http https]).freeze
+  SVG_REGEXP = /\A<svg .*>/i.freeze
```

Comment:
this is not going to work in all cases, as SVGs can have the XML tags at the start:

```xml
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg>
```

and still be valid. also They are also still valid if they start with a newline:

```xml

<svg ... />
```

It's probably better to just check if the svg tag is in there :D

Reply:
Changed ✅

<img width="497" alt="Screenshot 2020-10-02 at 13 26 07" src="https://user-images.githubusercontent.com/2786819/94918466-ebb30580-04b2-11eb-97d9-8855f3d9afe9.png">